atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/NameComponentDescription.java

```diff
@@ -20,71 +20,79 @@
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
 
-import org.apache.commons.lang.Validate;
+import org.apache.commons.lang3.Validate;
 
 import com.google.common.collect.Sets;
 import com.palantir.atlasdb.protos.generated.TableMetadataPersistence;
 import com.palantir.atlasdb.protos.generated.TableMetadataPersistence.LogSafety;
-import com.palantir.atlasdb.protos.generated.TableMetadataPersistence.NameComponentDescription.Builder;
 import com.palantir.atlasdb.protos.generated.TableMetadataPersistence.ValueByteOrder;
 
 @Immutable
-public class NameComponentDescription {
+public final class NameComponentDescription {
```

Comment:
Technically this is a dev break (for people who have subclassed `NameComponentDescription`). I think this is fine, but we might want to call it out in the release notes.